plugins/sio/dagens.py

```diff
@@ -5,18 +5,18 @@
 from plugins.sio.sio import Dagens
 
 
-@respond_to(r'^dagens$')
-@respond_to(r'^dagens (.*)')
-@listen_to(r'^!dagens$')
-@listen_to(r'^!dagens (.*)')
+@respond_to(r'(?i)^dagens\s*$')
```

Comment:
personlig så liker jeg at alle kommandoene er case sensitive, men det er nå meg.

Men utenom det så ser det helt greit ut for min del